src/components/artemis/documentation/index.md

```diff
@@ -1,7 +1,31 @@
 ---
-layout: iframe
-title: Apache ActiveMQ Artemis Documentation
-iframe: docs
+layout: default_md
+title: ActiveMQ Artemis Documentation
 title-class: page-title-artemis
+type: artemis
```

Comment:
Probably worth adding a redirect from the removed 'docs' page in case anyone was linking straight to it. 
I think something like follows would work (though in this case, might get away with just the 'docs'?) with the basic jekyll redirect, or you could add a .htaccess rule.
```
redirect_from:
- /components/artemis/documentation/docs